modules/db/src/main/java/org/opencastproject/db/Activator.java

```diff
@@ -137,6 +139,36 @@ else if (pooledDataSource.getMaxIdleTime() == 0) {
     logger.info("Database connection pool established at {}", jdbcUrl);
     logger.info("Database connection pool parameters: max.size={}, min.size={}, max.idle.time={}",
       pooledDataSource.getMaxPoolSize(), pooledDataSource.getMinPoolSize(), pooledDataSource.getMaxIdleTime());
+    Statement statement = pooledDataSource.getConnection().createStatement();
+
+    long time = System.currentTimeMillis();
+    String tableName = "oc-temp" + time;
```

Comment:
Nitpicking ;-P
To be consistent with the other table names:

```suggestion
    String tableName = "oc_temp_" + time;
```